mathics/builtin/arithmetic.py

```diff
@@ -1283,8 +1290,12 @@ def test(self, expr) -> bool:
 
 class RealSign(Builtin):
     """
-    <url>:WMA link:https://reference.wolfram.com/language/ref/RealAbs.html</url>
-
+    <url>
+        :Signum:
```

Comment:
The link name should be what Wikipedia calls it, so this should be "Sign function". Wikipedia mentions that this is also called "Signum"

Reply:
Ok, I can change this in another iteration.

Reply:
I have included that change in #886, which is the next round of patches.